libvips/include/vips/colour.h

```diff
@@ -93,8 +93,12 @@ extern "C" {
 #define VIPS_D3250_Y0 (100.0)
 #define VIPS_D3250_Z0 (45.8501)
 
+/* Note: constants align with those defined in lcms2.h, expect for
```

Comment:
Should be "except" I think.

Reply:
Ah, the infamous typo I always make :sweat_smile:. Thanks!